adsApp/src/adsAsynPortDriver.cpp

```diff
@@ -608,7 +608,7 @@ void adsAsynPortDriver::bulkReadThread()
                     asynPrint(asynTraceUser, ASYN_TRACE_ERROR,
                               "%s:%s: bulk read for %s (%d) failed\n",
                               driverName, functionName, paramInfo->drvInfo, j);
-                    continue;
+                    exit(1);
```

Comment:
As someone very new to this codebase, when do we `exit(-1)` vs `exit(1)`?  Do they have different behavior or do we just get different return codes out?  

Also just to confirm: the rebooting behavior is not in this package, but rather in how we handle the iocs themselves?  As written, this just closes the IOC?

Reply:
From what I'm aware of C++ only has two types of `exit()` values--0 or 1. When it's 0 it means the program has terminated successfully without any errors or interruptions. When it's 1 it means the program terminated abruptly with an error. `exit()` is used to terminate the currently running function.

`exit(-1)` seems to be legacy code. Any exit() function with a number other than 0 means the program terminated with an error. I found a list of exit codes in Linux. https://www.agileconnection.com/article/overview-linux-exit-codes  I don't think these are commonly used in modern C++.